src/TriggerCandidateMakerSupernova.cpp

```diff
@@ -29,20 +29,18 @@ TriggerCandidateMakerSupernova::operator()(const TriggerActivity& activity, std:
     std::chrono::time_point<std::chrono::steady_clock> now;
 
     // set all bits to one (probably this will mean down the line: "record every part of the detector")
-    uint32_t detid = 0xFFFFFFFF; // NOLINT(build/unsigned)
+    detid_t detid = ~0x0; // NOLINT(build/unsigned)
```

Comment:
Perhaps a `constexpr detid_t WHOLE_DETECTOR = ~0x0` in a header could be used here